client-js/app/elements/labrad-plot.ts

```diff
@@ -103,6 +104,9 @@ export class Plot extends polymer.Base {
 
   @property({type: String, value: 'zoomRect'})
   mouseMode: string;
+  
+  @property({type: Object, value: ''})
```

Comment:
True. I get confused as to how javascript (and typescript) call "objects" when I think of them as dictionaries. I would think that deps is an array of dictionaries, but Array seems fine here.